services/movie.py

```diff
@@ -0,0 +1,44 @@
+import init_django_orm  # noqa: F401
+from django.db.models import QuerySet
+
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+    if not genres_ids and not actors_ids:
```

Comment:
just use filter and use only one return

  if genres_ids:
        queryset = queryset.filter(genres__id__in=genres_ids)
    if actors_ids:
        queryset = queryset.filter(actors__id__in=actors_ids)
    return queryset